vscode/src/extension.ts

```diff
@@ -441,10 +440,10 @@ export function activate(context: ExtensionContext): VSNetBeansAPI {
             if (typeof res === 'string') {
```

Comment:
Please add the prompt message on line 402 for localisation: `"Input the directory path where the new file will be generated"`